README.md

```diff
@@ -50,9 +50,28 @@ flowchart TB
     end
 ```
 
-## Usage in a cartridge application
+## Usage in a Tarantool 3 application
+
+You need to install the Tarantool 3.0.2+ or 3.1+.
+
+1. Add a dependency to your application rockspec.
+2. Enable `roles.sharded-queue-router` role on all sharding `router` instances.
+3. Enable `roles.sharded-queue-storage` role on all sharding `storage`
+   isntances.
+4. Configure tubes for both roles.
+5. Do not forget to bootstrap the vshard in your application. See
+   [router.lua][./router.lua] as an example.
+
+You could see a full example of the configuration in the [config.yaml][./config.yaml].
 
-1. Add dependency to your application rockspec
+Be careful, it is unable to create or drop tubes dynamically by API calls
```

Comment:
Btw, why so?

Reply:
The problem here: we need to hold persistently information about tubes somewhere and to spread the information about it across the cluster.

From my point of view a reliable way to do this is to get information from the cluster configuration (the Cartridge updates the clusterwide config on `create_tube` as example). But maybe there are other ways.